roles/usegalaxy-eu.bashrc/defaults/main.yml

```
@@ -0,0 +1,4 @@
+---
+user_name: "{{ galaxy_user.name }}"
```

I think its good to use more specific variable names, to avoid unintended overwriting of other variables and less confusion in the vars files. e.g.
```suggestion
bashrc_user_name: "{{ galaxy_user.name }}"
```

---

All tasks in a `block` should be moved out to another tasks file and then should be included in the `main.yml`  where we can loop. I can implement this since you don't like the idea of invoking the role for every user.

---

I like that more, because the variables can stay in a vars file and are not in the playbook. Thank you!

---

Here is the update https://github.com/usegalaxy-eu/infrastructure-playbook/pull/610/commits/3b6594fbc950b531e163bf5ecc3ad4d6daf43865